includes/Utilities/Plugin_Request_Utility.php

```diff
@@ -134,4 +134,28 @@ public static function destroy_runner() {
 
 		static::$runner = null;
 	}
+
+	/**
+	 * Checks if the current request is a CLI request for the Plugin Checker.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool Returns true if is a CLI request for the plugin check else false.
+	 */
+	public static function is_plugin_check() {
```

Comment:
Instead of a new method in this general class (which is not specific to CLI at all), I suggest we use the `CLI_Runner::is_plugin_check()` method instead and make that static.